components/GroupView/GroupView.js

```diff
@@ -72,20 +85,22 @@ export default class GroupView extends Component {
             <Text>Group Members</Text>
             {userList}
           </View>
+            {userGroups}
         </Content>
         <Footer>
           <FooterTab>
-            <Button onPress={this._handleChangePage}>
-              <Text>Next Page</Text>
+            <Button onPress= {()=> this._handleChangePage('Default')}>
```

Comment:
This seems fine. Watch out for spaces after () in arrow functions. Is the linter not picking this up?